src/ReTestItems.jl

```diff
@@ -161,6 +183,7 @@ function runtests(
     logs::Symbol=default_log_display_mode(report, nworkers),
     verbose_results::Bool=logs!=:issues && isinteractive(),
 )
+    nworker_threads = _validate_nworker_threads(nworker_threads)
```

Comment:
reading this, i'd think this _only_ does argument validation, but it also converts the input to a String if necessary... can we rename it to make that clear?

maybe even just
```suggestion
    nworker_threads = _validated_nworker_threads(nworker_threads)
```
but maybe there's something clearer still?

Reply:
Good idea:+1: